argobots.spec

```diff
@@ -109,6 +111,10 @@ rm -f %{buildroot}%{_libdir}/*.{l,}a
 %doc README
 
 %changelog
+* Tue Apr 02 2024 Brian J. Murrell <brian.murrell@intel.com> - 1.2-1
```

Comment:
Let's update the datestamp here to be current.

Reply:
- [ ] Update time stamp

Reply:
Fixed with commit 69e02db